src/core/datatypes/trees/TopologyNode.h

```diff
@@ -158,13 +158,15 @@ namespace RevBayesCore {
         void                                        removeAllChildren(void);                                                            //!< Removes all of the children of the node
         size_t                                      removeChild(TopologyNode* c);                                                       //!< Removes a specific child
         void                                        removeTree(Tree *t);                                                                //!< Removes the tree pointer
+        void                                        resolveMultifurcation(bool resolve_root = true);                                    //!< If node has more than 2 children, randomly resolve them into a bifurcating subtree
         void                                        setAge(double a, bool propagate = true );                                           //!< Set the age of this node (should only be done for tips).
         void                                        setBranchLength(double b, bool flag_dirty=true);                                    //!< Set the length of the branch leading to this node.
         void                                        setIndex(size_t idx);                                                               //!< Set the index of the node
 
         void                                        setName(const std::string& n);                                                      //!< Set the name of this node
         void                                        setNumberOfShiftEvents(size_t n);                                                   //!< Set the number of shift events for stochastic character maps
         void                                        setParent(TopologyNode* p, bool recompute_branch_length = true);                    //!< Sets the node's parent
+        void                                        setParentAge(double minbl);                                                         //!< Sets the age of the parent to the age of the oldest of its children + minbl
```

Comment:
this seems misleading - the function actually sets the age of the node, not the parent ?

Reply:
See above; fixed.